pages/schedule.re

```diff
@@ -4,220 +4,158 @@ module Link = ReactRouterDom.Link;
-      </h2>
-      <dl className=style##entries>
-        {
-          Data.Timetable.day1Timetable
-          |> List.map(createRow)
-          |> Array.of_list
-          |> ReasonReact.array
-        }
-      </dl>
-      <h2 id="day2">
-        <time dateTime="2018-05-12">
-          {"12" |> s}
-          <sup> {"th" |> s} </sup>
-          {" May, Saturday at " |> s}
-          <Link to_="/attendees/#conference-venue">
-            {"TU Wien Kuppelsaal" |> s}
-          </Link>
-        </time>
-      </h2>
-      <dl className=style##entries>
-        {
-          Data.Timetable.day2Timetable
-          |> List.map(createRow)
-          |> Array.of_list
-          |> ReasonReact.array
-        }
-      </dl>
-      <h2 id="day3">
-        <time dateTime="2018-05-13">
-          {"13" |> s}
-          <sup> {"th" |> s} </sup>
-          {" May, Sunday at " |> s}
-          <Link to_="/attendees/#workshop-and-hackathon-venue">
-            {"ImpactHub" |> s}
-          </Link>
-        </time>
-      </h2>
-      <dl className=style##entries>
-        <dt className=style##entryTime>
-          <time> {"9:00 - 10:00" |> s} </time>
-        </dt>
-        <dd className=style##entryDescription>
-          {"Doors open & forming Teams for the Hackathon" |> s}
-        </dd>
-        <dt className=style##talkTime>
-          <time> {"10:00 - 13:00" |> s} </time>
-        </dt>
-        <dd className=style##talkDescription>
-          {
-            {js|
-Choose between:
-
-1. **Open Mic Session** – everyone can come and give a talk. Speakers can opt-in to be recorded and their talk published by our ReasonConf channel.
-2. **Hackathon** – team up with others or work alone on your project of choice
-3. **Vienna Tour** – explore the city center with one of our local volunteers
-            |js}
-            |> md
-          }
-        </dd>
-        <dt className=style##entryTime>
-          <time> {"13:00 - 14:00" |> s} </time>
-        </dt>
-        <dd className=style##entryDescription> {"Lunch" |> s} </dd>
-        <dt className=style##talkTime>
-          <time> {"14:00 - 17:30" |> s} </time>
-        </dt>
-        <dd className=style##talkDescription>
-          {
-            {js|
-Choose between:
-
-1. **Hackathon** – team up with others or work alone on your project of choice
-2. **Vienna Tour** – explore the city center with one of our local volunteers
-            |js}
-            |> md
-          }
-        </dd>
-        <dt className=style##entryTime>
-          <time> {"18:00 - 22:00" |> s} </time>
-        </dt>
-        <dd className=style##entryDescription>
-          {"Get together at a local wine tavern (Heuriger)" |> s}
-        </dd>
-      </dl>
-    </section>,
+    <div>
+      <h1> "Schedule"->s </h1>
+      <p>
+        "All workshops will be held on the first day from 9:30am to about 5pm."
+        ->s
+      </p>
+      <h2> "Workshops"->s </h2>
+      <h3> "Beginner Reason & ReasonReact Workshop"->s </h3>
+      <p>
```

Comment:
I think that’s the problem. Don’t wrap markdown into `p` tag.

Reply:
@okonet markdown creates `p` tags as well (I think you got the problem later)

Reply:
That's why I said don't wrap markdown into `p`. This structure doesn't make any sense:

```
<p>
   <h3> <!-- this is coming from markdown --></h3>
  <p>content</p>
</p>
```